src/main/java/org/junitpioneer/jupiter/DisableIfNameExtension.java

```diff
@@ -24,25 +25,31 @@
 
 	@Override
 	public ConditionEvaluationResult evaluateExecutionCondition(ExtensionContext context) {
-		Optional<DisableIfName> disable = findAnnotation(context.getElement(), DisableIfName.class);
+		Optional<Method> testMethod = context.getTestMethod();
 
-		if (!disable.isPresent()) {
-			return enabled("No instructions to disable");
-		} else if (!context.getTestMethod().isPresent()) {
-			return enabled("Only disable at method level");
+		if (!testMethod.isPresent()) {
+			return enabled("Only disable at method level so the parameterized tests could be registered");
 		}
-		DisableIfName details = disable.get();
-		boolean toDisable;
-		if (details.regex()) {
-			toDisable = context.getDisplayName().matches(details.value());
-		} else {
-			toDisable = context.getDisplayName().contains(details.value());
+
+		Optional<DisableIfDisplayName> disableIf = findAnnotation(testMethod, DisableIfDisplayName.class);
```

Comment:
Please update to the new methods, provided by #187 after it is merged

Reply:
Will wait for #187 to be merged to resolve all the other comments then

Reply:
We just merged it, @nishantvas.